backend/src/routes/get.ts

```diff
@@ -0,0 +1,28 @@
+import { sql } from "@/lib/db";
+import { Elysia } from "elysia";
+
+export const getRoute = new Elysia({ prefix : '/get'} )
+  .get('/email', async ( { cookie } ) => {
+    const sessionID = cookie.auth_session.value;
+
+    try {
+      const email = await sql`
+        SELECT auth_user.email
+        FROM auth_user 
+        INNER JOIN user_session
+        ON user_session.user_id=auth_user.id
+        WHERE user_session.id=${sessionID};
+      `
+
+      return {
+        status : "success",
+        data : email
+      }
+    } catch ( error ) {
+      return {
+        status : "error",
+        message : "Error occured somewhere, please try again"
+      }
+    }
+
+  })
```

Comment:
What is this? Shouldn't check session api give you user data already?

Reply:
Shouldn't return ALL DATA to client, might be including hashed_password or sentitive information

Reply:
> > What is this? Shouldn't check session api give you user data already?
> 
> `check session api` is return only yes or no if the session is expire.

In this case, it wouldn't be valid state anyway when client's data is changed and we need to change it back to valid state. When login just save all the data once (return required data with `/sign-in`), and `checkSession()` only check if session is valid or not, not update data anymore